test/models/solution/mentor_discussion_post_test.rb

```diff
@@ -3,6 +3,12 @@
 class Iteration::DiscussionPostTest < ActiveSupport::TestCase
   include MarkdownFieldMatchers
 
+  test "generates uuid before create" do
```

Comment:
Is the "before create" part relevant to this test, or is it enough to just state that the uuid must be generated?

Reply:
Good point. I will remove the "before create".